shapefile/src/main/scala/geotrellis/shapefile/ShapeFileReader.scala

```diff
@@ -47,9 +48,10 @@ object ShapeFileReader {
 
   def readSimpleFeatures(path: String): Seq[SimpleFeature] = readSimpleFeatures(new URL(s"file://${new File(path).getAbsolutePath}"))
 
-  def readSimpleFeatures(url: URL): Seq[SimpleFeature] = {
+  def readSimpleFeatures(url: URL, charSet: Charset = Charset.forName("ISO-8859-1")): Seq[SimpleFeature] = {
```

Comment:
Hm, unfortunately, this breaks compatability, the signature of this function will be slighely different. 

I want to suggest to have here two functions:

```scala
def readSimpleFeatures(url: URL, charSet: Charset): Seq[SimpleFeature] = ??? // which will use the charset
// for the backwards compat
def readSimpleFeatures(url: URL): Seq[SimpleFeature] =  readSimpleFeatures(url, Charset.forName("ISO-8859-1"))
```

Reply:
I have made the change but want to wait before I sort out the ECA issues. I have signed the general ECA but not project specific agreement. Apparently, I cannot see the "Committer Tools" link on the right hand side that is described in the contributing documentation.